src/averageAges.js

```diff
@@ -15,11 +15,20 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  return getAvarage(
+    people
+      .filter(person => person.sex === 'm')
+      .filter(man => !century || getCentury(man) === century)
+      .map(person => person.died - person.born)
+  );
+}
```

Comment:
Very nice job with reusing same function for three cases.
However, it is a bad practice to operform unnecessary complex operations inside fuction call.
Store `people` array processing results in a variable and then pass it to your fuction. 
Do it for all three cases.